packages/components/menu/src/sub-menu.ts

```diff
@@ -256,6 +257,7 @@ export default defineComponent({
 
       timeout?.()
       ;({ stop: timeout } = useTimeoutFn(() => {
+        submenuRef.value?.focus()
```

Comment:
It is better to trigger before the animation.

```suggestion
      if (
        rootMenu.isMenuPopup &&
        event.type === 'mouseenter' &&
        rootMenu.props.menuTrigger === 'hover'
      ) {
        submenuRef.value?.focus()
        }
```

Reply:
Thanks for the review.
I have corrected the code.
Please confirm it.